README.md

```diff
@@ -11,19 +11,32 @@ Tested on Ubuntu 18.04, Jetson Boards and Windows 11 x64 Visual Studio 2017 Comm
 git clone --recursive https://github.com/Apra-Labs/ApraPipes.git
 ```
 
+### Prerequisites for CUDA 
+* Install [CUDA Toolkit 10.2](https://developer.nvidia.com/cuda-10.2-download-archive?target_os=Linux&target_arch=x86_64&target_distro=Ubuntu&target_version=1804)
+* Download [Cudnn](https://developer.nvidia.com/rdp/cudnn-archive#a-collapse765-102) and extract where cuda is installed.
+* Rename the file base/vcpk.json to base/vcpkg.json.bkp and base/vcpkg.cuda.json to base/vcpkg.json
```

Comment:
Can  you add this step in the bat file

Reply:
Had a discussion with @yashrajsapra , we will eventually have just two vcpkg.json one for cuda and one for no cuda. For now lets leave this as is.